components/filesystem.rst

```diff
@@ -236,6 +247,11 @@ Its behavior is the following::
     * if ``$path`` does not exist, it returns null.
     * if ``$path`` exists, it returns its absolute fully resolved final version.
 
+.. note::
+
+    If you wish to canonicalize the path without checking its existence, you can
```

Comment:
Reading this might be confusing if you don't know what "canonicalize the path" exactly means.